packages/nuget/Sentry.EntityFramework/4.2.1.json

```diff
@@ -4,5 +4,31 @@
   "version": "4.2.1",
   "package_url": "https://www.nuget.org/packages/Sentry.EntityFramework",
   "repo_url": "https://github.com/getsentry/sentry-dotnet",
-  "main_docs_url": "https://docs.sentry.io/platforms/dotnet/entityframework"
+  "main_docs_url": "https://docs.sentry.io/platforms/dotnet/entityframework",
+  "categories": [
+    "desktop",
+    "server"
```

Comment:
should work on mobile too I believe but this is an integration not a framework. do we also add that info to it?